app/src/examples/PerfomanceMonitorExample.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useRef, useState } from 'react';
+import { Text, StyleSheet, Pressable, View } from 'react-native';
+import { PerformanceMonitor } from 'react-native-reanimated';
+
+import EmptyExample from './EmptyExample';
+import BokehExample from './BokehExample';
+import PlanetsExample from './PlanetsExample';
+import EmojiWaterfallExample from './EmojiWaterfallExample';
+
+enum Examples {
+  Empty = 'Empty Example',
+  Bokeh = 'Bokeh Example',
+  Planets = 'Planets Example',
+  Emojis = 'Emoji Waterfall Example',
+}
+
+export default function PerformanceMonitorExample() {
+  const exampleElements = useRef(
+    new Map<Examples, JSX.Element>([
+      [Examples.Empty, <EmptyExample />],
+      [Examples.Bokeh, <BokehExample />],
+      [Examples.Planets, <PlanetsExample />],
+      [Examples.Emojis, <EmojiWaterfallExample />],
+    ])
+  );
+
+  const [currentExample, setCurrentExample] = useState(<EmptyExample />);
+
+  const setElementByExample = (example: Examples) => {
+    setCurrentExample(exampleElements.current.get(example)!);
+  };
+
+  return (
+    <>
+      <PerformanceMonitor />
+      {currentExample}
+      <View style={styles.buttonContainer}>
+        {[
+          Examples.Empty,
+          Examples.Bokeh,
+          Examples.Planets,
+          Examples.Emojis,
+        ].map((element) => (
+          <Pressable
+            key={element}
+            style={styles.button}
+            onPress={() => setElementByExample(element)}>
```

Comment:
According to previous comments, here I would suggest rendering example based solely on the state (not 100% sure if this compiles but you should get the idea):

```suggestion
      {exampleElements.current.get(currentExample)!}
      <View style={styles.buttonContainer}>
        {[
          Examples.Empty,
          Examples.Bokeh,
          Examples.Planets,
          Examples.Emojis,
        ].map((element) => (
          <Pressable
            key={element}
            style={styles.button}
            onPress={() => setCurrentExample(element)}>
```